R/recode_into.r

```diff
@@ -0,0 +1,109 @@
+#' @title Recode values from one or more variables into a new variable
+#' @name recode_into
+#'
+#' @description
+#' This functions recodes values from one or more variables into a new variable.
+#' It is a convenient function to avoid nested [`ifelse()`] statements.
```

Comment:
We could say it's similar to `dplyr::case_when()`, it'll be easier to understand what it does